packages/components/CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - React: `'use client';` directive is applied on all components for main and `ssr` sub-package
   ([#2654](https://github.com/porsche-design-system/porsche-design-system/pull/2654))
 
+#### Fixed
+
+- Fixes regression in `observeChildren` with nested components
```

Comment:
Probably would be good to mention the bug in nested p-tabs, otherwise consumers won't know what this is about.